ignite/metrics/precision.py

```diff
@@ -127,18 +127,6 @@ def thresholded_output_transform(output):
             default, CPU.
 
     """
-
-    def __init__(
```

Comment:
I think this was kept for proper docs rendering. Can you please check that ?

Reply:
Reverted the change! The Precision and Recall docstrings render like before now.